File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] output=new int[2];
        for(int i=0;i<nums.length;i++){
            for(int j=i;j<nums.length;j++){
                if((nums[i]+nums[j]==target)&&(i!=j)){
                    output[0]=i;
                    output[1]=j;
                }
            }
        }
        return output;
    }
}

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:

        for l in range(len(nums)-1):
            r=l+1
            while r<len(nums):
                if nums[l]==nums[r]:
                    return nums[l]
                r+=1

                    

        